### PR TITLE
feat: switch to atlas for db model

### PR DESCRIPTION
**Pull request by moshloop**

### CODE REVIEW

**Comment by yashmehrotra** [comment]
@moshloop This is great 👍🏽

File: functions/functions.go [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package functions
+
+import "embed"
+
+//go:embed *.sql
+var functions embed.FS
+
+var Functions map[string]string
+
+func GetFunctions() (map[string]string, error) {
```
Comment (yashmehrotra) [inline functions/functions.go:10]: How is this being called ?

**Review by yashmehrotra — APPROVED** [review]

**Comment by yashmehrotra** [comment]
@moshloop For triggers, we might have to do a `DROP IF EXISTS ... CREATE TRIGGER` in a always kind of a script

File: functions/functions.go (hunk 1 above)
Comment (moshloop) [reply functions/functions.go:10]: https://github.com/flanksource/incident-commander/pull/218/files#diff-953a885e28970388a5ee8146da2db0488f6c2539fe2db2caca45a4033f090985R138

**Pull request merged**